pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -129,13 +129,19 @@ def render(self, task) -> RenderableType:
     class MetricsTextColumn(ProgressColumn):
         """A column containing text."""
 
-        def __init__(self, trainer, pl_module):
+        def __init__(self, trainer):
             self._trainer = trainer
-            self._pl_module = pl_module
             self._tasks = {}
             self._current_task_id = 0
+            self.metrics = {}
             super().__init__()
 
+        def update(self, metrics):
```

Comment:
Btw, do you know why  mypy is not triggering here? The configuration shows we have rich (and other bars) included. 
cc @carmocca

Reply:
Maybe we need to update it to `"pytorch_lightning.callbacks.progress.*"`?